### ISSUE TITLE
Bio.Alphabet has been removed from Biopython

### ISSUE THREAD

**Issue by Tgrandis** [HadrienG/InSilicoSeq#233]
I just installed InSilicoSeq and tried it out. I got warnings that some packages were missing (SciPy etc) but after installing those I got an error of a different kind. I have Python 3.10.10 on a Linux Mint machine.

Traceback (most recent call last):
  File "/home/cab/miniconda3/bin/iss", line 6, in <module>
    from iss.app import main
  File "/home/cab/miniconda3/lib/python3.10/site-packages/iss/app.py", line 8, in <module>
    from iss import generator
  File "/home/cab/miniconda3/lib/python3.10/site-packages/iss/generator.py", line 12, in <module>
    from Bio.Alphabet import IUPAC
  File "/home/cab/miniconda3/lib/python3.10/site-packages/Bio/Alphabet/__init__.py", line 20, in <module>
    raise ImportError(
ImportError: Bio.Alphabet has been removed from Biopython. In many cases, the alphabet can simply be ignored and removed from scripts. In a few cases, you may need to specify the ``molecule_type`` as an annotation on a SeqRecord for your script to work correctly. Please see https://biopython.org/wiki/Alphabet for more information.

**Comment by jtclaypool** [comment]
Not sure if this will work for you but I solved this through the pip upgrade. The conda install wasn't working for me. Also, make sure you have the appropriate versions installed 
```pip install --upgrade InSilicoSeq```

```
install_requires=['numpy', 'scipy', 'biopython<=1.78',
                      'pysam>=0.15.1', 'future',
                      'joblib', 'requests']```

**Comment by HadrienG** [comment]
This was fixed in v1.5.1
